online-shop/src/app/add-new/add-new.component.ts

```diff
@@ -0,0 +1,68 @@
+import {Component, OnInit} from '@angular/core';
+import {Location} from "@angular/common";
+import {ActivatedRoute} from "@angular/router";
+import {FormGroup, Validators, FormBuilder} from "@angular/forms";
+import {Product} from "../models/products";
+import {ProductService} from "../services/product-service";
+
+@Component({
+  selector: 'app-add-new',
```

Comment:
When you name your components, you should give more context. If you have more entities, how would somebody know that this is for products and not for orders?